### ISSUE TITLE
sphinx doc error

### ISSUE THREAD

**Issue by rwst** [bluescarni/piranha#15]
```
ralf@ark:~/piranha/doc/sphinx> make html
sphinx-build -b html -d _build/doctrees   . _build/html
Making output directory...
Running Sphinx v1.2.3

Exception occurred:
  File "conf.py", line 96, in <module>
    import sphinx_rtd_theme
ImportError: No module named sphinx_rtd_theme
The full traceback has been saved in /tmp/sphinx-err-vuNiw2.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://bitbucket.org/birkenfeld/sphinx/issues/>. Thanks!
Makefile:45: recipe for target 'html' failed
make: *** [html] Error 1
```
resolved via:
```
diff --git a/doc/sphinx/conf.py b/doc/sphinx/conf.py
index db9d5a3..539fec2 100644
--- a/doc/sphinx/conf.py
+++ b/doc/sphinx/conf.py
@@ -92,10 +92,10 @@ pygments_style = 'sphinx'
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-#html_theme = 'sphinxdoc'
-import sphinx_rtd_theme
-html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
+html_theme = 'sphinxdoc'
+#import sphinx_rtd_theme
+#html_theme = "sphinx_rtd_theme"
+#html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
 
 
 # Theme options are theme-specific and customize the look and feel of a theme
```

**Comment by bluescarni** [comment]
@rwst Thanks for the report. I use the RTD theme for the sphinx docs which are uploaded here:

http://bluescarni.github.io/piranha/

It would probably be better to wrap the import of ``sphinx_rtd_theme`` in a try/except block, and use the default html theme if RTD is not available. Do you think this would be an acceptable solution?

```python
try:
    import sphinx_rtd_theme
    html_theme = "sphinx_rtd_theme"
    html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
except ImportError:
    print('The sphinx RTD theme was not found, using the default \'sphinxdoc\' theme.')
    html_theme = 'sphinxdoc'
```

**Comment by rwst** [comment]
@bluescarni while I have no direct experience with Sphinx this looks fine to me. On a different subject, do  you know of open software that is already using Piranha polynomials/series? I need an example to see how it's done.

**Comment by bluescarni** [comment]
@rwst There's this automatic differentiation project which is based on Piranha:

https://github.com/darioizzo/audi

There are some example of usage in the performance tests, eg:

https://github.com/bluescarni/piranha/blob/master/tests/audi_perf.cpp
https://github.com/bluescarni/piranha/blob/master/tests/fateman1.hpp
https://github.com/bluescarni/piranha/blob/master/tests/gastineau2.hpp

symengine is planning to use Piranha in their polynomial module, as far as I know, but I don't think they have actually started using it yet.

Do you have any specific use case in mind? Feel free to ask me here or send me a message to bluescarni at the google mail domain. I know user documentation is sorely lacking at the moment, sorry about that.

**Comment by bluescarni** [comment]
@rwst this should be fixed in #34.
